src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,16 @@
+<hibernate-configuration xmlns="http://www.hibernate.org/xsd/orm/cfg">
+    <session-factory>
+        <property name="connection.driver.class">org.hibernate.hsqldb.jdbcDriver</property>
+        <property name="connection.driver.class">com.mysql.cj.jdbc.Driver</property>
+        <property name="connection.url">jdbc:mysql://localhost:3306/cinema_app</property>
+        <property name="connection.username">root</property>
+        <property name="connection.password">oiY39Pnq#H71gJ&amp;</property>
+        <property name="dialect">org.hibernate.dialect.MySQLDialect</property>
+        <property name="show_sql">true</property>
+        <property name="hbm2ddl.auto">create-drop</property>
+
+        <mapping class="mate.academy.hibernate.relations.model.Actor"/>
+        <mapping class="mate.academy.hibernate.relations.model.Country"/>
+        <mapping class="mate.academy.hibernate.relations.model.Movie"/>
+    </session-factory>
+</hibernate-configuration>
```

Comment:
always make sure your file ends with one empty line, otherwise this red symbol will appear on GitHub